server/server.go

```diff
@@ -500,7 +501,7 @@ git push --set-upstream origin %[3]s
 git checkout master
 `, tag, repo, branch)
 		url := fmt.Sprintf("https://github.com/mattermost/mattermost-marketplace/compare/production...%s?quick_pull=1&labels=3:+QA+Review,2:+Dev+Review", branch)
-		msg = fmt.Sprintf("Plugin was successfully signed and uploaded to Github and S3.\nTag: **%s**\nRepo: **%s**\n[Release Link](%s)\nTo add this release to the Plugin Marketplace run inside your local Marketplace repository:\n```%s\n```\nUse %s to open a Pull Request.", tag, repo, releaseURL, marketplaceCommand, url)
+		msg = fmt.Sprintf("Plugin was successfully signed and uploaded to Github and S3.\nTag: **%s**\nRepo: **%s**\nCommitSHA: **%s**\nForce: **%v**\n[Release Link](%s)\nTo add this release to the Plugin Marketplace run inside your local Marketplace repository:\n```%s\n```\nUse %s to open a Pull Request.", tag, repo, commitSHA, force, releaseURL, marketplaceCommand, url)
```

Comment:
If no `commitSHA` is provided, can the commit sha of `master` get printed instead?

Reply:
Printing the getting the sha of `master` is surely option. But could you omit the `CommitSHA:` if no `commitSHA` is provided.